f5_openstack_agent/lbaasv2/drivers/bigip/monitor_manager.py

```diff
@@ -444,9 +448,15 @@ def update_member_status_task(self):
                             m['device_info']['password']
                         )
 
-                        # TODO(nik) add v6 later when the config item is added
+                        if self.use_mgmt_ipv6 and m['mgmt_ipv6']:
+                            hostname = m['mgmt_ipv6']
+                            if ":" in hostname and \
+                               not hostname.startswith("["):
+                                hostname = "[" + hostname + "]"
+                        else:
```

Comment:
if this bigip only has one ipv6 addr, we will have to use ipv6 addr, even if it is not preferred.